src/libprojectM/MilkdropPresetFactory/Expr.cpp

```diff
@@ -1415,10 +1415,15 @@ Expr *Expr::jit(Expr *root, std::string name)
     std::vector<Type *> arg_typess;
     arg_typess.push_back(IntegerType::get(Context,32));
     arg_typess.push_back(IntegerType::get(Context,32));
-    Constant* c = jitx.module->getOrInsertFunction<Type*>("Expr_eval",
+#if LLVM_VERSION_MAJOR >= 10
```

Comment:
I would be fine with requiring a recent version since LLVM JIT is an optional feature anyway. We need to do some work to use the newer interface though, see #360

Reply:
That makes sense.